public/components/trace_analytics/components/traces/span_detail_flyout.tsx

```diff
@@ -307,9 +308,9 @@ export function SpanDetailFlyout(props: {
         props.startTime
       },to:${props.endTime}))&_q=(filters:!(),query:(dataset:(dataSource:(id:'${
```

Comment:
TODO the timestamps are currently undefined, need to pass them into props from parent